### PR TITLE
Updates README; converts organizers.yml to organizers.json

### PR DESCRIPTION
**Pull request by yflicker**

### CODE REVIEW

**Comment by yflicker** [comment]
@oestrich would you mind just reading the top of the README and make sure I didn't write anything blatantly incorrect? the part about getting this repo running locally — don't worry about the JSON definitions.

File: README.md [hunk 1]
```diff
@@ -1,48 +1,66 @@
...
+This README assumes that the reader has previous knowledge and experience using Ruby on Rails, jekyll, GitHub pages, JSON, and related web dev technologies.
+
+To edit the website locally you must first:
+1. Get Ruby on Rails running on your computer
+1. Make sure you have bundler. You can try: `gem install bundler`
+1. Clone this repository
+1. Make sure you have all the latest gems, e.g. via `bundle exec install`
+
+To review edits you make before committing them:
 1. jekyll serve
```
Comment (oestrich) [inline README.md]: I would use `bundle exec jekyll serve`

File: README.md [hunk 2]
```diff
@@ -1,48 +1,66 @@
 # HackBaltimore
 
-Get Ruby on Rails running on your computer
+The HackBaltimore website is statically generated using jekyll and GitHub pages.
 
-1. bundle exec install
+This README assumes that the reader has previous knowledge and experience using Ruby on Rails, jekyll, GitHub pages, JSON, and related web dev technologies.
+
+To edit the website locally you must first:
+1. Get Ruby on Rails running on your computer
```
Comment (oestrich) [inline README.md]: We don't need rails for this. Just the later steps.

Comment (yflicker) [reply README.md]: so just ruby is needed?

Comment (oestrich) [reply README.md]: Yeah, good point. Just ruby, bundler, then bundle install should get it going.

**Pull request merged**